CONFIGURATION.md

```diff
@@ -102,6 +102,11 @@ further below for information on that.
 # Default: 'required'.
 #presenterField = 'required'
 
+# Specifies if the series field should be 'hidden', 'optional' or 'required'.
+# Default: 'required'.
+#seriesField = 'required'
```

Comment:
I guess `required` is the most common use case.

Reply:
Oh wow good catch! So the code defaults to `optional`, so these docs are incorrect. Probably copy&paste error on my side... for some reason. I would actually argue to make it `optional` by default, as currently implemented in code. Will push that change.